_layouts/base.html.slim

```diff
@@ -93,6 +93,12 @@ body.antialiased
       - unless page.url.start_with? '/news/'
         .large-4.columns.secondary
           - if page.url =~ /^\/docs\/.+/
+            - if page.toc
```

Comment:
@mojavelinux Is there an easy way to generate TOC (HTML) from an Asciidoctor::Document ?